package.json

```diff
@@ -83,7 +83,7 @@
     "@types/sinon-chai": "^3.2.4",
     "@types/yargs": "^17.0.28",
     "chai": "^4.2.0",
-    "chai-as-promised": "^8.0.0",
+    "chai-as-promised": "^7.1.1",
```

Comment:
I downgraded this due to this issue: https://github.com/chaijs/chai-as-promised/issues/301#issuecomment-2160108977